etl/steps/data/garden/news/2024-05-08/guardian_mentions.py

```diff
@@ -25,35 +46,86 @@ def run(dest_dir: str) -> None:
         df=tb,
         countries_file=paths.country_mapping_path,
     )
-    tb = geo.harmonize_countries(
-        df=tb,
-        countries_file=paths.country_mapping_path,
-    )
 
     ## Get relative values
-    tb = add_relative_indicator(tb, "num_pages_tags")
-    tb = add_relative_indicator(tb, "num_pages_mentions")
+    tb = add_relative_indicators(tb, ["num_pages_tags", "num_pages_mentions"])
 
-    tb = tb.format(["country", "year"])
+    ## Add per-capita indicators
+    tb = geo.add_population_to_table(tb, ds_population)
+    for column in ["num_pages_tags", "num_pages_mentions"]:
+        tb[f"{column}_per_million"] = tb[column] / tb["population"] * 1_000_000
+    tb = tb.drop(columns="population")
 
+    # Estimate 10-year average
+    tb_10y_avg = tb[(tb["year"] >= 2014) & (tb["year"] <= 2023)].copy()
```

Comment:
Maybe set these as global values at the top so it's easy to update in the future?